src/com/jalasoft/search/model/Search.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Search.java
+ *
+ * Copyright (c) 2018 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.jalasoft.search.model;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ Class Searches Files by content, extension, name, and some other attributes receives a Object Criteria, evaluates it and return and Object.
+ *
```

Comment:
Please avoid unnecessary spaces in the comments

Reply:
all of them Removed